src/app/services/main-form-manager/config-export-helper.ts

```diff
@@ -873,7 +877,7 @@ export class ConfigExportHelper {
                     consultedItemEvent: unmanagedRenderFields.consultedItemEvent,
                     actionOnItemEvent: unmanagedRenderFields.actionOnItemEvent,
                     globalActionEvent: unmanagedRenderFields.globalActionEvent,
-                    useColorService: !!widgetData.renderStep.useColorService,
+                    useColorService: true,
```

Comment:
Always true ?

Reply:
For the moment I think we can set this value to true because it's only used in grid mode for the color of the Title and the circle icon, and the builder don't generate grid mode. We will change that when the builder will be able to build config in grid mode

Reply:
👌